Docs/deployment/configuration.md

```diff
@@ -22,12 +22,12 @@ Two configuration methods can be used to control Santa: a local configuration pr
 | BannedBlockMessage            | String     | This is the message shown to the user when a binary is blocked because of a rule if that rule doesn't provide a custom message. If this is not configured a reasonable  default is provided. |
 | ModeNotificationMonitor       | String     | The notification text to display when the client goes into Monitor mode. Defaults to "Switching into Monitor mode". |
 | ModeNotificationLockdown      | String     | The notification text to display when the client goes into Lockdown mode. Defaults to "Switching into Lockdown mode". |
-| SyncBaseURL*                  | String     | The base URL of the sync server.         |
```

Comment:
nice catch - the * used to mean something different before we moved to mobileconfigs.